webpack/assets/javascripts/react_app/common/colors.scss

```diff
@@ -1 +1,7 @@
 $pf-black-200: #ededed;
+$pf-red-200: #a30000;
```

Comment:
can we use this from the patternfly package? @sharvit ?

Reply:
When my css pr will merge you would be able to import variables:
https://github.com/theforeman/foreman/pull/5212/files#diff-f430da9fad5e785cb1f815fd719fdc5aR1